NtupleProducer/python/HiggsTauTauProducer_102X.py

```diff
@@ -337,6 +337,52 @@
         finalCut = cms.string(' '),
 )
 
+
+################################
+## Boosted Tau Reco & ID bug-fix for Legacy samples
+# see: https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuidePFTauID#Running_of_the_bug_fix_version_o
+################################
+from RecoTauTag.Configuration.boostedHPSPFTaus_cff import ca8PFJetsCHSprunedForBoostedTaus
+process.ca8PFJetsCHSprunedForBoostedTausPAT = ca8PFJetsCHSprunedForBoostedTaus.clone(
+    src=cms.InputTag("packedPFCandidates"),
+    jetCollInstanceName = cms.string('subJetsForSeedingBoostedTausPAT')
+)
+cleanedBoostedTau = cms.EDProducer("PATBoostedTauCleaner",
+                                   src = cms.InputTag('slimmedTausBoosted'),
+                                   pfcands = cms.InputTag('packedPFCandidates'),
+                                   vtxLabel= cms.InputTag('offlineSlimmedPrimaryVertices'),
+                                   ca8JetSrc = cms.InputTag('ca8PFJetsCHSprunedForBoostedTausPAT','subJetsForSeedingBoostedTausPAT'),
+                                   removeOverLap = cms.bool(True),
+                               )
+setattr(process, "cleanedSlimmedTausBoosted", cleanedBoostedTau)
+
+updatedBoostedTauName = "slimmedBoostedTausNewID" #name of pat::Tau collection with new tau-Ids
+
+import RecoTauTag.RecoTau.tools.runBoostedTauIdMVA as boostedTauIdConfig
+boostedTauIdEmbedder = boostedTauIdConfig.BoostedTauIDEmbedder(process, cms, debug = False,
+                                                        updatedTauName = updatedBoostedTauName,
+                                                        PATTauProducer = cms.InputTag('cleanedSlimmedTausBoosted'),
+                                                        srcChargedIsoPtSum = cms.string('chargedIsoPtSumNoOverLap'),
+                                                        srcNeutralIsoPtSum = cms.string('neutralIsoPtSumNoOverLap'),
+                                                        toKeep = [ #choose tauIDs to be rerun
+                                                            #"2017v2", "dR0p32017v2", "newDM2017v2", #classic MVAIso tau-Ids
+                                                            #"deepTau2017v1", #deepTau Tau-Ids
+                                                            #"DPFTau_2016_v0", #D[eep]PF[low] Tau-Id
+                                                            "2017v2","deepTau2017v1","againstEle2018" # default in twiki
```

Comment:
For non-boosted taus we use `deepTau2017v2p1` as deepTau version, is the recommendation differend for boosted taus?

Reply:
This is what appears in the POG twiki 'bugfix' code snippet, but I'm not sure it is an actual recommendation. I'll try to make sure of it and change it if needed

Reply:
I did not find any specific treatment for boosted taus, so I switched back to ```deepTau2017v2p1```.

Reply:
To be 100% sure you sould check with Konstantin which is the correct tauID for boosted taus.

Reply:
Discussion ongoing with Konstantin: MVA ID should be used for boosted taus. There are apparently some other tricky points to consider, that are potential show-stopper for some. I'll comment with details in a detached comment.